apps/src/lib/levelbuilder/lesson-editor/LevelTokenDetails.jsx

```diff
@@ -93,69 +49,9 @@ class LevelTokenDetails extends Component {
     activityPosition: PropTypes.number.isRequired,
 
     //redux
-    levelKeyList: PropTypes.object.isRequired,
-    levelNameToIdMap: PropTypes.objectOf(PropTypes.number).isRequired,
-    chooseLevel: PropTypes.func.isRequired,
-    addVariant: PropTypes.func.isRequired,
-    removeVariant: PropTypes.func.isRequired,
-    setActiveVariant: PropTypes.func.isRequired,
-    setLevelField: PropTypes.func.isRequired,
     setScriptLevelField: PropTypes.func.isRequired
   };
 
-  containsLegacyLevel() {
-    return this.props.scriptLevel.levels.some(level =>
-      /^blockly:/.test(this.props.levelKeyList[level.id])
-    );
-  }
-
-  handleLevelSelected = (variant, levelId) => {
-    this.props.chooseLevel(
-      this.props.activityPosition,
-      this.props.activitySectionPosition,
-      this.props.scriptLevel.position,
-      variant,
-      levelId
-    );
-  };
-
-  handleAddVariant = () => {
-    this.props.addVariant(
```

Comment:
this method would need to be changed to use the add level dialog to add a variant. I think this where "fixing forward" would get complicated.